CMakeLists.txt

```diff
@@ -2,19 +2,19 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 
 SET(FC "/usr/bin/gfortran" CACHE STRING "Fortran Compiler")
```

Comment:
I am not sure this is necessary - doesn't CMake automatically find an appropriate Fortran compiler? If you want to make sure it's `gfortran`, I'd prefer not to encode the path to make it more portable.